build/configs/rtl8721csm/loadable_apps/defconfig

```diff
@@ -247,9 +247,9 @@ CONFIG_ARCH_BOARD_HAVE_FLASH=y
 #
 CONFIG_FLASH_PARTITION=y
 CONFIG_FLASH_MINOR=0
-CONFIG_FLASH_PART_SIZE="16,8,1024,1024,5608,"
-CONFIG_FLASH_PART_TYPE="none,none,kernel,kernel,smartfs,"
-CONFIG_FLASH_PART_NAME="km0_bl,km4_bl,kernel,kernel,userfs,"
+CONFIG_FLASH_PART_SIZE="16,8,16,472,1024,1024,5608,"
```

Comment:
In PR#4671(https://github.com/Samsung/TizenRT/pull/4671), we changed the smartfs partition size as `5608`, because of reserved for secure storage.
So if you fix the secure storage size as 16K and 472K, we can fix the smartfs partition size also.
Please calculate the remained size and change the smartfs partition size also.

Reply:
So in my calculation, Bootloader and Secure Storage is 512KB, so 2 Kernel 1024 = 2048, then smartfs will be 5440.
16,8,16,472,1024,1024,5440,", is my calculation correct?

Reply:
Yes. It looks okay.

Reply:
> Please split the commit between defconfigs and code changes.

But there are related, can I split it? the code change will modify the start address for Secure Storage.